pkg/cmd/admin/top/graph.go

```diff
@@ -0,0 +1,149 @@
+package top
+
+import (
+	"github.com/golang/glog"
+	gonum "github.com/gonum/graph"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	"github.com/openshift/origin/pkg/api/graph"
+	kubegraph "github.com/openshift/origin/pkg/api/kubegraph/nodes"
+	imageapi "github.com/openshift/origin/pkg/image/api"
+	imagegraph "github.com/openshift/origin/pkg/image/graph/nodes"
+)
+
+const (
+	ImageLayerEdgeKind          = "ImageLayer"
+	ImageTopLayerEdgeKind       = "ImageTopLayer"
+	ImageStreamEdgeKind         = "ImageStream"
+	HistoricImageStreamEdgeKind = "HistoricImageStream"
+	PodImageEdgeKind            = "PodImage"
+	ParentImageEdgeKind         = "ParentImage"
+
+	// this is empty layer check sum from schema1
+	emptyLayerSHA256 = "sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4"
```

Comment:
I must have been looking through wrong piece of code, because I haven't noticed that one over there. fixed.